src/librehatti/settings.py

```diff
@@ -53,13 +53,7 @@
 
 MEDIA_URL = ''
 
-STATIC_ROOT = ''
-
-STATIC_URL = '/librehatti/static/'
-
-STATICFILES_DIRS = (
-    '../../static',
-)
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')                                                  STATIC_URL = '/librehatti/static/'                                                              STATICFILES_DIRS = [                                    os.path.join(BASE_DIR, '../static')     ]
```

Comment:
Looks like `STATIC_ROOT`, `STATIC_URL` and `STATICFILES_DIRS` are all in one line. Separate it.

Reply:
Corrected in https://github.com/GreatDevelopers/LibreHatti/pull/524